packages/app/src/app/pages/Dashboard/Components/Folder/FolderCard.tsx

```diff
@@ -38,7 +38,7 @@ export const FolderCard: React.FC<FolderItemComponentProps> = ({
       width: '100%',
       height: 240,
       backgroundColor: 'grays.700',
-      border: '1px solid',
+      border: 'transparent',
```

Comment:
Should we add borders and higher contrast for the focus-within state?

![Screen Shot 2022-09-15 at 11 33 51](https://user-images.githubusercontent.com/24959348/190431682-c1e08d90-72f0-4183-a814-99eb5acb713f.png)

Using the box shadow is okay but the contrast ratio of 1.12:1 between the foreground and background fails to meet the minimum focus appearance criteria. Reference here: https://www.w3.org/WAI/WCAG22/Understanding/focus-appearance-minimum.html

Reply:
This will be address in other PR after merged the colors

Reply:
We are replace the border to background but I need the colors to do that.